packages/elvis/CHANGELOG.json

```diff
@@ -13,12 +13,25 @@
             "Deprecated the classes <code>.e-alert__content</code>, <code>.e-alert__links</code>, <code>.e-alert--actions</code> and <code>.e-alert--local</code>.",
             "Deprecated the class <code>.e-input--placeholder</code>."
           ],
-          "components": [{ "displayName": "Alert", "url": "https://design.elvia.io/components/alert" },{"displayName": "Text Field", "url": "https://design.elvia.io/components/input"}],
+          "components": [
+            { "displayName": "Alert", "url": "https://design.elvia.io/components/alert" },
+            { "displayName": "Text Field", "url": "https://design.elvia.io/components/input" }
+          ],
           "fixes": [
             "Simply remove the classes <code>.e-alert__content</code>, <code>.e-alert--actions</code> and <code>.e-alert--local</code>, since they are no longer required. Replace <code>.e-alert__links</code> with <code>.e-alert__actions</code>",
             "Simply remove the class <code>.e-input--placeholder</code>, since it is no longer required."
           ]
```

Comment:
Not related to this PR, but: Should we consider splitting these two breaking changes into two different objects in the changelog-list? Even though the type is called "breaking_changeS", I feel like each change should be it's own entry (at least at component level).

Reply:
I agree, however previously we've always just had one entry of each kind (one new_features, one patch, etc.), so I'm not 100% sure if the changelog search (and other systems) in web actually supports searching through that kind of structure 🤔